web/ASC.Web.Editor/src/helpers/useMfScripts.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import {
+  loadComponent,
+  useDynamicScript,
+} from "../components/DynamicComponent";
+import {
+  STUDIO_SCOPE,
+  FILES_SCOPE,
+  STUDIO_REMOTE_ENTRY_URL,
+  FILES_REMOTE_ENTRY_URL,
+} from "../helpers/constants";
+
+function useMfScripts() {
+  const [isInitialized, setIsInitialized] = React.useState(false);
+  const [isError, setIsError] = React.useState(false);
+
+  const { ready: filesReady, failed: filesFailed } = useDynamicScript({
+    id: FILES_SCOPE,
+    url: FILES_REMOTE_ENTRY_URL,
+  });
+  const { ready: studioReady, failed: studioFailed } = useDynamicScript({
+    id: STUDIO_SCOPE,
+    url: STUDIO_REMOTE_ENTRY_URL,
+  });
+
+  React.useEffect(async () => {
```

Comment:
Fix this "React Hook Warnings for async function in useEffect: useEffect function must return a cleanup function or nothing"